en/css/README.md

```diff
@@ -130,7 +129,7 @@ Your file should now look like this:
 
         {% for post in posts %}
             <div>
-                <p>published: {{ post.published_date }}</p>
+                <p>Published: {{ post.published_date }}</p>
```

Comment:
Is there a reason to change the capitalization of "published:"? (I'm not saying it shouldn't be changed, I'm simply wondering why it shall be changed.)

Reply:
No reason. It looked cleaner. I'll remove all those changes of "published:".

Reply:
Reverted changes in https://github.com/DjangoGirls/tutorial/pull/1681/commits/6f899d338caa430613984553ae065cfc69465496